src/vs/base/common/async.ts

```diff
@@ -6,12 +6,12 @@
 import { CancellationToken, CancellationTokenSource } from './cancellation.js';
 import { BugIndicatingError, CancellationError } from './errors.js';
 import { Emitter, Event } from './event.js';
+import { Lazy } from './lazy.js';
```

Comment:
Sorting of imports seems irrelevant to this task. Not everyone in the team sorts imports and generally I think we should either sort all imports in the project and have a rule for it or not sort at all.

Reply:
Ugh I think this was caused by format on save.